packages/docs/src/routes/docs/overview/index.mdx

```diff
@@ -51,17 +51,10 @@ Qwik is a new kind of web framework that can deliver instant loading web applica
   </div>
   <div class="card">
     <h3>Instant-on</h3>
-    <p>
-      Unlike other frameworks, Qwik is resumable which means Qwik applications require 0 hydration.
-      This allows Qwik apps to have instant-on interactivity, regardless of size or complexity
-    </p>
+    <p>Unlike other frameworks, Qwik is resumable which means Qwik applications require 0 hydration. This allows Qwik apps to have instant-on interactivity, regardless of size or complexity</p>
```

Comment:
with a line break after the `<p>` tag the following error was thrown when running the docs locally:
```
10:07:06 PM [vite] Internal server error: Expected a closing tag for `<p>` (61:5-61:8) before the end of `paragraph`
  Plugin: vite-plugin-qwik-city
  File: /Users/zanettin/development/qwik/qwik/packages/docs/src/routes/docs/overview/index.mdx:60:5
  58 |    </div>
  59 |    <div class="card">
  60 |      <h3>Optimized for speed</h3>
     |       ^
  61 |      <p>Qwik has unprecedented performance, offering sub-second full page loads even on mobile
  62 |        devices. Qwik achieves this by delivering pure HTML, and incrementally loading JS only
```